app/components/works/show/files_component.html.erb

```diff
@@ -1,22 +1,23 @@
-<table class="table table-sm mb-5" id="filesTable">
-  <thead class="table-light">
-  <tr>
-    <th scope="col" class="table-title">Files
-      <span class="ms-3">
-        <%= helpers.turbo_frame_tag dom_id(work, 'edit_file'), src: edit_button_work_path(work, tag: 'file'), target: '_top' %>
-      </span>
-      <% if download_all? %>
-        <%= link_to work_zip_path(work), class: 'ms-4', data: { turbo: false } do %>
-          <span class="fa-solid fa-download" aria-hidden="true"></span><span class="visually-hidden">Download All Files</span>
+<turbo-frame id="<%= dom_id(work, 'files_list') %>" src="<%= files_list_work_path(work) %>" loading="lazy">
+  <table class="table table-sm mb-5" id="filesTable">
+    <thead class="table-light">
+    <tr>
+      <th scope="col" class="table-title">Files
+        <span class="ms-3">
+          <%= helpers.turbo_frame_tag dom_id(work, 'edit_file'), src: edit_button_work_path(work, tag: 'file'), target: '_top' %>
```

Comment:
I'd be inclined to omit edit and download all until the frame is fully rendered.

Reply:
I removed the icons. I'm seeing it does cause the "Description" and "Hide?" headers to shift once the icons appear.